PagerState.message is private

**Describe the bug**

I want to add a bit change to `DefaultInputClassifier` to customize keybinding.
So I copied `DefaultInputClassifier` to my `CustomInputClassifier`.
But it can't be compiled.

**To Reproduce**

The error message is below:

```
error[E0616]: field `message` of struct `PagerState` is private
  --> src/custom_input_classifier.rs:46:23
   |
46 |                 if ps.message.is_some() {
   |                       ^^^^^^^ private field
```

**Expected behavior**

Changing `message` to `public` or adding an API which has the same behavior as `message.is_some()`.

--- a/src/state.rs
+++ b/src/state.rs
@@ -53,7 +53,7 @@ pub struct PagerState {
     /// Any message to display to the user at the prompt
     /// The first element contains the actual message, while the second element tells
     /// whether the message has changed since the last display.
-    pub(crate) message: Option<String>,
+    pub message: Option<String>,
     /// The prompt that should be displayed to the user, formatted with the
     /// current search index and number of matches (if the search feature is enabled),
     /// and the current numbers inputted to scroll

